src/main.rs

```diff
@@ -120,7 +120,7 @@ fn create_app<'a, 'b>(
                 .help(
                     "User defined memory zone parameters \
                      \"size=<guest_memory_region_size>,file=<backing_file>,\
-                     shared=on|off,hugepages=on|off\"",
+                     shared=on|off,hugepages=on|off,host_numa_nodes=<nodes_bitmask>\"",
```

Comment:
IMO the bitmask input is a terrible user experience. Could we support a `host_numa_nodes=n,m,p` instead and then convert that to a bitmask when actually making the syscall?

Reply:
Ok I'll do that!